R/description.R

```diff
@@ -145,40 +145,6 @@ renv_description_parse_field <- function(field) {
 
 }
 
-renv_description_remotes <- function(descpath) {
```

Comment:
Removed because it's basically a duplicate of `renv_project_remotes_description_remotes()`